static/app/components/profiling/flamegraphOptionsContextMenu.tsx

```diff
@@ -39,12 +45,86 @@ interface FlameGraphOptionsContextMenuProps {
   isHighlightingAllOccurences: boolean;
   onCopyFunctionNameClick: () => void;
   onHighlightAllOccurencesClick: () => void;
+  profileGroup: ProfileGroup | null;
+}
+
+function isSupportedPlatformForGitHubLink(platform: string | undefined): boolean {
+  if (platform === undefined) {
+    return false;
+  }
+
+  return /node|python/.test(platform);
```

Comment:
Is regex necessary here?

Reply:
Looking at the platform keys, it looks like we have node-* and python-* and I just want to enable it for all of those.

Reply:
I believe this platform is coming from the profile, not the project so it should be limited to `node` and `python`.

Reply:
Arf, you are right. I looked at the platform key type and got confused